### ISSUE TITLE
Consistent naming for Berkeley

### ISSUE THREAD

**Issue by mbroecheler** [thinkaurelius/titan#864]
sometimes referred to as berkeleyje (shorthand) and sometimes berkeleydb (conf files).

**Comment by dalaro** [comment]
@mbroecheler do you have a preference?

Though I defer to your preference, I would go with berkeleyje in the absence of one:
- removes the potential for ambiguity between the pure-java "Java Edition" and the Java-language binding for the native edition
- it's slightly more common on aureliusgraphs, according to google search result count estimates for each term in double quotes:
  
  ```
  Results: about 287 for "berkeleyje"
  
  Results: about 238 for "berkeleydb"
  ```
- continuing to recognize the -je storage.backend shorthand, as opposed to adding -db and keeping -je alongside it for backwards compat, might reduce confusion about whether they are identical or distinct for new users going through search results or different doc versions

**Comment by dalaro** [comment]
I think d343091d610c7392aa2a57bc814de488420f5548 converted all of the important uses of "berkeleydb" to "berkeleyje".  There are still some bits of prose in the docs that refer to "BerkeleyDB" without the "Java Edition" qualifier, but file paths and classnames should be converted.

I looked for instances of "berkeleydb" with these commands executed from the root:
- `find -iname '*berkeleydb*'`
- `grep -Iir 'berkeleydb' */src docs`

I also tweaked README.textile in the root.
